opencv/pom.xml

```diff
@@ -70,41 +71,47 @@
+              </excludes>
             </configuration>
           </execution>
           <execution>
             <id>${javacpp.platform}${javacpp.platform.extension}</id>
             <phase>package</phase>
             <configuration>
               <excludes>
-                <exclude>${javacpp.platform.library.path}/lib/*.so*</exclude>
-                <exclude>${javacpp.platform.library.path}/sdk/**/*.so*</exclude>
-                <exclude>${javacpp.platform.library.path}/sdk/native/jni/include/**</exclude>
-                <exclude>${javacpp.platform.library.path}/**/*cascade*</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/*.exp</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/*.lib</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/*.obj</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/bin/*</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/lib/*.a</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/lib/*.dylib</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/lib/*.so*</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/sdk/**/*.a</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/sdk/**/*.dylib</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/sdk/**/*.so*</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/share/**/*.a</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/share/**/*.dylib</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/share/**/*.so*</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/x86/**/*.lib</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/x86/**/*.dll</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/x86/**/*.exe</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/x64/**/*.lib</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/x64/**/*.dll</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/x64/**/*.exe</exclude>
-                <exclude>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/**/*cascade*</exclude>
+                <exclude>lib/*.so*</exclude>
```

Comment:
I don't think we need to change any of those directories anymore. Could you try to revert all those changes back?

Reply:
Right. Done.

Reply:
It's still the same. Let me know when you've reverted the paths here.

Reply:
Can you either revert the changes to these excludes, or remove them if you find that they are no longer necessary?

Reply:
Reverted